src/pyadjoint/adjoint_source.py

```diff
@@ -140,6 +146,41 @@ def _write_specfem(self, buf, time_offset):
 
         np.savetxt(buf, to_write)
 
+    def write_to_asdf(self, ds, time_offset):
+          """
+          Writes the adjoint source to an ASDF file.
+          Note: For now it is assumed SPECFEM will be using the adjoint source
+
+          :param ds: The ASDF data structure read in using pyasdf
+          :type ds: str
```

Comment:
I added some documentation on this and the lat/long/ele arguments.

On Oct 21, 2015, at 3:28 AM, Lion Krischer <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/pyadjoint/adjoint_source.py<https://github.com/krischer/pyadjoint/pull/1#discussion_r42591051>:

> @@ -140,6 +146,41 @@ def _write_specfem(self, buf, time_offset):
>
>          np.savetxt(buf, to_write)
>
> +    def write_to_asdf(self, ds, time_offset):
> +          """
> +          Writes the adjoint source to an ASDF file.
> +          Note: For now it is assumed SPECFEM will be using the adjoint source
> +
> +          :param ds: The ASDF data structure read in using pyasdf
> +          :type ds: str


Can you please also document the time_offset parameter?

—
Reply to this email directly or view it on GitHub<https://github.com/krischer/pyadjoint/pull/1/files#r42591051>.